src/main/java/org/terasology/funnyblocks/system/BouncySystem.java

```diff
@@ -0,0 +1,67 @@
+package org.terasology.funnyblocks.system;
+
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.Time;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.logic.characters.CharacterImpulseEvent;
+import org.terasology.logic.characters.CharacterMoveInputEvent;
+import org.terasology.logic.characters.CharacterMovementComponent;
+import org.terasology.logic.location.LocationComponent;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.registry.In;
+import org.terasology.world.BlockEntityRegistry;
+import org.terasology.world.WorldProvider;
+import org.terasology.entitySystem.systems.RegisterMode;
+
+import org.terasology.funnyblocks.component.BouncyBlockComponent;
+
+@RegisterSystem(RegisterMode.AUTHORITY)
+public class BouncySystem extends BaseComponentSystem implements UpdateSubscriberSystem 
+{
+
+	@In
+	private BlockEntityRegistry blockEntityProvider;
+
+	@In
+	private Time time;
+
+	@In
+	private EntityManager entityManager;
+
+	@In
+	private WorldProvider worldProvider;
+
+	@Override
+	public void update(float delta) 
+	{
+
+	}
+	
+	private static final Logger logger = LoggerFactory.getLogger(BouncySystem.class);
```

Comment:
This variable should be the top class variable (`private static` comes before plain `private`). In general https://github.com/MovingBlocks/Terasology/wiki/Checkstyle will help you spot code style issues like that :-)